packages/node-integration-tests/suites/tracing/auto-instrument/mongodb/test.ts

```diff
@@ -2,6 +2,9 @@ import { MongoMemoryServer } from 'mongodb-memory-server';
 
 import { assertSentryTransaction, getEnvelopeRequest, runServer } from '../../../../utils';
 
+// mongodb-memory-server may download a MongoDB binary in different Node versions
```

Comment:
which node versions does this apply to?

Reply:
also - lol

Reply:
Thought it was just `16`, but it looks like it installs for `12`, `14` and `16`. Maybe it's better to move this out to pre-test.

Reply:
> Maybe it's better to move this out to pre-test.

I think thats a good idea